imports/wasi_snapshot_preview1/poll_test.go

```diff
@@ -61,17 +62,6 @@ func Test_pollOneoff_Errors(t *testing.T) {
 	mod, r, log := requireProxyModule(t, wazero.NewModuleConfig())
 	defer r.Close(testCtx)
 
-	// We aren't guaranteed to have a terminal device for os.Stdin, due to how
-	// `go test` forks processes. Instead, we test if this is consistent. For
-	// example, when run in a debugger, this could end up true.
-	// See also `terminal_test.go`.
-	expectedFdReadErr := wasip1.ErrnoNotsup
-	if stat, err := os.Stdin.Stat(); err != nil {
-		if stat.Mode()&fs.ModeCharDevice != 0 {
-			expectedFdReadErr = wasip1.ErrnoBadf
-		}
-	}
-
```

Comment:
this test alone should make less sense because now we can simulate the different states of the device